### PR TITLE
Make platform-tools setup customizible and tell it to mono/sdks setup.

### PR DESCRIPTION
**Pull request by atsushieno**
Context: https://github.com/mono/mono/pull/9473#issue-200303674

### CODE REVIEW

File: Configuration.props [hunk 1]
```diff
@@ -84,6 +84,7 @@
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
     <XABuildToolsVersion>28</XABuildToolsVersion>
     <XABuildToolsFolder Condition="'$(XABuildToolsFolder)' == ''">28.0.0</XABuildToolsFolder>
+    <XAPlatformToolsVersion>28.0.0</XAPlatformToolsVersion>
```
Comment (jonpryor) [inline Configuration.props]: Should this be `Conditional`/overridable, as `$(XABuildToolsFolder)` is?

**Comment by jonpryor** [comment]
This PR should also include a bump to external/mono, otherwise this change won't actually control anything.

Or we do the mono bump as a separate PR...

**Comment by jonpryor** [comment]
Separate mono bump PR: #1981

File: Configuration.props (hunk 1 above)
Comment (atsushieno) [reply Configuration.props]: That's true. Though there are other properties without Condition, they are added in some need perhaps? We'll do so too.

**Pull request merged**